### DISCUSSION TITLE
Trabalhar com sessões

### DISCUSSION THREAD

**Discussion by eazschornack** [HashLoad/horse#243] [Q&A]
Alguém teria um exemplo pra trabalhar com as sessões do horse?
Gostaria de criar uma classe pra cada sessão conectada.

Existe uma forma de fazer isso, sem trabalhar com JWT e headers?

**Comment by viniciussanchez** [comment]
Sim existe sim... você pode salvar os dados na Req.Session

**Reply by viniciussanchez** [reply]
E não precisa usar o JWT. Qualquer coisa pode criar seu próprio middleware para interceptar as requisições e salvar isso se quiser. Ou entao no próprio callback da request.
